packages/salesforcedx-vscode-apex/src/settings.ts

```diff
@@ -13,3 +13,7 @@ export function retrieveTestCodeCoverage(): boolean {
     .getConfiguration(SFDX_CORE_CONFIGURATION_NAME)
     .get<boolean>('retrieve-test-code-coverage', false);
 }
+
+export const enableSyncInitJobs: boolean = vscode.workspace
```

Comment:
This should be wrapped in a function like `retrieveTestCodeCoverage`.  Otherwise doesn't it run when the module it loaded?

Reply:
also we should write a unit test for it